Ginger/Ginger/BusinessFlowPages_New/AddActionMenu/MainAddActionsNavigationPage.xaml.cs

```diff
@@ -72,15 +75,27 @@ private void NavPnlActionFrame_ContentRendered(object sender, EventArgs e)
         {
             if ((sender as Frame).Content == null)
             {
-                (sender as Frame).Visibility = Visibility.Collapsed;
-                xNavigationBarPnl.Visibility = Visibility.Collapsed;
-                xAddActionsOptionsPnl.Visibility = Visibility.Visible;
+                if(applicationModelView)
+                {
+                    (sender as Frame).Visibility = Visibility.Collapsed;
+                    xNavigationBarPnl.Visibility = Visibility.Visible;
+                    xAddActionsOptionsPnl.Visibility = Visibility.Collapsed;
+                    xApplicationModelsPnl.Visibility = Visibility.Visible;
+                }
+                else
+                {
+                    (sender as Frame).Visibility = Visibility.Collapsed;
+                    xNavigationBarPnl.Visibility = Visibility.Collapsed;
+                    xAddActionsOptionsPnl.Visibility = Visibility.Visible;
+                    xApplicationModelsPnl.Visibility = Visibility.Collapsed;
```

Comment:
all of this APIModel View need to be in the dedicated page for Application Models and not here in main page